lib/zstd/common/debug.c

```diff
@@ -22,6 +22,4 @@
 
 #include "debug.h"
 
-#if (DEBUGLEVEL>=2)
```

Comment:
Revert this

Reply:
Changed the base of my branch to the previous commit to avoid this issue, thanks for spotting!